### DISCUSSION TITLE
More detailed example of Headless?

### DISCUSSION THREAD

**Discussion by arkmech** [wobsoriano/svelte-sonner#105] [Q&A] [answered]
So when I pass a custom component to toast.custom.

Can I pass props to the component?

toast.custom(<MyComponent title=“Hello World” />)

possible?

**Comment by wobsoriano** [comment, marked as answer]
This is how you would use a custom component with props:

```ts
toast.custom(MyComponent, {
	componentProps: { title: 'Hello World' }
});
```

**Reply by arkmech** [reply]
Great, thanks! after playing with this. Do you plan on exposing the Component primitives? As realizing, I'm taking for granted all the aria and accessibility with the main component.
